toolboxes/gtplus/workflow/gtPlusISMRMRDReconWorker3DTSPIRIT.h

```diff
@@ -442,7 +450,7 @@ performUnwrapping(gtPlusReconWorkOrder3DT<T>* workOrder3DT, const hoNDArray<T>&
             if ( workOrder3DT->coilMap_->get_number_of_elements() > 0 )
             {
                 if ( performTiming_ ) { gt_timer3_.start("permute coil map RO to 4th dimension ... "); }
-                GADGET_CHECK_RETURN_FALSE(Gadgetron::permuteROTo4thDimensionFor3DRecon(*workOrder3DT->coilMap_, *coilMapPermuted));
+                GADGET_CHECK_RETURN_FALSE(this->permuteROTo4thDimensionFor3DRecon(*workOrder3DT->coilMap_, *coilMapPermuted));
```

Comment:
I'm going to stop writing the same thing over and over, but this should be a generic call.